### DISCUSSION TITLE
Adding Google Fonts to Webpack?

### DISCUSSION THREAD

**Discussion by katherinevfry** [nss-evening-cohort-14/evening-client-side#150] [Help Me!] [answered]
Your name:
Katy Fry

Have you searched the other help tickets? 
I have!

Describe your issue
How would I go about adding Google Fonts to webpack? Is this the best option for using different fonts in my project? In the past I know we'd put the source tags in the HTML, but I know that's probably not the best practice with Webpack. I wanted to check here for best practices before totally messing everything up with my setup. 

What have you googled?
https://www.npmjs.com/package/google-fonts-webpack-plugin
https://chriscourses.com/blog/loading-fonts-webpack
https://stackoverflow.com/questions/42708257/google-fonts-webpack

**Comment by drteresavasquez** [comment, marked as answer]
So what you'd do is use the @import statement that you get from the font and add it to one of your sass files. Here is an example:

https://github.com/drteresavasquez/black-history-trivia/blob/9414a9dd86f2270ef8a4b730b445fb8ceb8eb762/src/styles/globals/_fonts.scss#L3

You could also download the font and include it in your project, but for these, the import should be fine. 

You can find the import statement once you select a font. Just remove the style tags when using it in your sass file. Here is an example of the import on google fonts:
![image](https://user-images.githubusercontent.com/29741570/109422402-ffe72a00-79a0-11eb-8c9b-cd357d8602fc.png)

**Reply by katherinevfry** [reply]
Thanks, Dr. T! This is perfect!!

**Reply by drteresavasquez** [reply]
@katherinevfry you're welcome!
